src/responder/nss/nsssrv_netgroup.c

```diff
@@ -428,7 +428,7 @@ static void set_netgr_lifetime(uint32_t lifetime,
 
     tv = tevent_timeval_current_ofs(lifetime, 0);
     te = tevent_add_timer(step_ctx->nctx->rctx->ev,
-                          step_ctx->nctx->gctx, tv,
+                          netgr, tv,
```

Comment:
Why did you decide to use `netgr` as a context?

 IMHO it is cleaner to use `struct nss_ctx` instead of `struct getent_ctx`